src/helpers/Monitor.cpp

```diff
@@ -1240,7 +1240,7 @@ void CMonitor::setSpecialWorkspace(const PHLWORKSPACE& pWorkspace) {
     }
 
     g_pEventManager->postEvent(SHyprIPCEvent{"activespecial", pWorkspace->m_szName + "," + szName});
-    g_pEventManager->postEvent(SHyprIPCEvent{"activespecialv2", pWorkspace->m_iID + "," + pWorkspace->m_szName + "," + szName});
+    g_pEventManager->postEvent(SHyprIPCEvent{"activespecialv2", std::to_string(pWorkspace->m_iID) + "," + pWorkspace->m_szName + "," + szName});
```

Comment:
Previously, the C++ compiler was interpreting this as `","[pWorkspace->m_iID]` . I think this would either result in garbage or the event being identical to the v1 activespecial event. I don't think this should be left as undefined behavior, but we could either solidify it as identical to the v1 event or merge this code which fixes it to what I assume was intended.

Reply:
ye